lib/gemfile_helper.rb

```diff
@@ -19,8 +19,33 @@ def load_dotenv
                               )
     end
 
+    GEM_NAME = '[A-Za-z0-9\.\-\_]+'.freeze
+    GEM_OPTIONS = '(.+?)\s*(?:,\s*(.+?)){0,1}'.freeze
+    GEM_SEPARATOR = '\s*(?:,|\z)'.freeze
```

Comment:
It's not needed, spaces are not valid in the GEM_NAME expression they will be `.scan` will skip over them and start the next match when any character that is allowed in GEM_NAME is found. In the [specs](https://github.com/cantino/huginn/pull/1366/files/d830459341337eba121208a0afbf96fba434e5b2#diff-5f806d1ddd6f1709c07247893609d1e7R22) gem are separated with spaces.